meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,130 @@
+# v0.29.0 release changelogs
+
+In this release, we integrate a really demanded feature: the possibility to custom the relevancy during the search. We also made a lot of improvements regarding the indexing speed.
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), the official documentation is not updated yet, but will be ready for the official v0.29.0 release. The work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Matching customization
+
+It's now possible to control the relevancy behavior during the search by choosing to return only documents containing ALL the query words.
+
+How to activate it?
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "american ninja", "matchingStrategy": "all" }'
+```
+
+Here Meilisearch will return only documents containing "american" AND "ninja".
+
+By default, Meilisaearch still returns documents containing all the query words, then documents that don't contain all the query words.
+
+<!-- The following line should only be put in the official release changelogs, not for the RC -->
+Check out the [documentation]().
+
+Made by @ManyTheFish in #2578 and meilisearch/milli#598.
+
+## Thai language support
+
+The Thai language is now supported 🇹🇭
+
+If your language is missing in Meilisearch, please consider contributing to the [Meilisearch tokenizer](https://github.com/meilisearch/charabia) by following the CONTRIBUTING.md file and integrating your intended normalizer/segmenter.
+
+The whole core team is available to help you contribute. Feel free to ask for any clarifications you need!
+
+Made by our dear contributor @aFluffyHotdog and by @ManyTheFish in meilisearch/charabia#114, meilisearch/charabia#115, meilisearch/milli##611, and #2636.
+
+## Performance improvements for indexing
```

Comment:
WDYT to explain the soft deletion and short-circuit permitting to not re-index the same document?

Reply:
Hum not sure I want to make the README heavier for something we want to make transparent for the users. I would rather add metrics when we will have ones 😄 
However I will link your issues

Reply:
Agree with @curquiza on this one.